Build fails with cargo install frauth on windows

Building on Windows via cargo install frauth fails:
`error: expected token: `,`
  --> ~\.cargo\registry\src\github.com-1ecc6299db9ec823\frauth-0.2.0\src\util.rs:26:9
   |
26 |         PATHS.user_info.display()
   |         ^^^^^ expected `,`

error: aborting due to previous error`

this can be circumvented, by adding a "," after line 25, however at least with stable-x86_64-pc-windows-gnu it than fails with a linker error for curl (which probably is an issue with libcurl_sys and not this package):
	
[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(url.o):url.c:(.text$Curl_init_userdefined+0xa): undefined reference to `__imp___acrt_iob_func'

[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(cookie.o):cookie.c:(.text$Curl_cookie_init+0x27b): undefined reference to `__imp___acrt_iob_func'

[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(cookie.o):cookie.c:(.text$Curl_flush_cookies+0x17f): undefined reference to `__imp___acrt_iob_func'

[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(formdata.o):formdata.c:(.text$Curl_getformdata.part.0+0x266): undefined reference to `__imp___acrt_iob_func'

[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(setopt.o):setopt.c:(.text$Curl_vsetopt+0x368e): undefined reference to `__imp___acrt_iob_func'

[..]\deps\libcurl_sys-94a1f16a4b8bdb84.rlib(mprintf.o):mprintf.c:(.text$curl_mprintf+0x29): more undefined references to `__imp___acrt_iob_func' follow

--- a/cli/src/util.rs
+++ b/cli/src/util.rs
@@ -22,7 +22,7 @@ pub fn create_private_file(path: &Path) -> Result<File> {
     // TODO: Figure out how file permissions work on Windows, or link to an issue
     #[cfg(not(unix))]
     eprintln!(
-        "Warning! You should set the permissions for {} to only be readable by this user!"
+        "Warning! You should set the permissions for {} to only be readable by this user!",
         PATHS.user_info.display()
     );
 

